deps/ccommon/rust/ccommon_rs/src/buf.rs

```diff
@@ -0,0 +1,292 @@
+// ccommon - a cache common library.
+// Copyright (C) 2018 Twitter, Inc.
```

Comment:
Incorrect year. I think we can make the license header the shorter version. eg:
```
// Copyright 2019 Twitter, Inc.
// Licensed under the Apache License, Version 2.0
// http://www.apache.org/licenses/LICENSE-2.0
```

Defer to @thinkingfish on whether it's appropriate to use the even shorter header.

Reply:
I've fixed the date for this. Will replace the license header if we decide to go with the shorter one.